samples/cpp/beam_search_causal_lm/README.md

```diff
@@ -18,6 +18,7 @@ optimum-cli export openvino --trust-remote-code --model TinyLlama/TinyLlama-1.1B
 `beam_search_causal_lm TinyLlama-1.1B-Chat-v1.0 "Why is the Sun yellow?"`
 
 To enable Unicode characters for Windows cmd open `Region` settings from `Control panel`. `Administrative`->`Change system locale`->`Beta: Use Unicode UTF-8 for worldwide language support`->`OK`. Reboot.
+Also, you can enable UTF-8 mode by setting environment variable `PYTHONIOENCODING="utf8"`.
```

Comment:
Describe why would someone want to enable it. Also add an error message that is solved by `PYTHONIOENCODING`.

Reply:
Please, add it to every sample